noir-projects/noir-contracts/compile.sh

```diff
@@ -0,0 +1,82 @@
+echo "app_subscription_contract"
```

Comment:
Why has this appeared here?

Reply:
Ah this was just a simple script I created because I could not get the contracts to not compile in parallel,shouldn't be part of the PR.